src/qfactors.jl

```diff
@@ -0,0 +1,80 @@
+## Computation of Q-factors to provide an estimate of the RV precision for a given spectrum as a function of SNR
+
+## Making the approximation that the spectrum as an ``average'' SNR per pixel when estimating the RV precision
+"""
+    precRV_fromQ(Q,SNR,Npix;c = 299792458)
+
+Compute the RV precision from the Q factor, the SNR per pixel, and the number of pixels in the spectrum.
+
+# Arguments
+- `Q::Real`: Q factor of the spectrum
+- `SNR::Real`: SNR per pixel of the spectrum
+- `Npix::Real`: Number of pixels in the spectrum
+
+# Keyword Arguments
+- `c::Real=299792458`: Speed of light in m/s
```

Comment:
You can use `Korg.c_cgs` (unqualified) for this.

Reply:
I will stop passing this as a keyword then and do Korg.c_cgs*1e-2